### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#504] [Esami & Esoneri M-Z]
### Traccia

<img width="874" alt="image" src="https://github.com/user-attachments/assets/a7678918-eadc-4a36-bc50-7811234617f3">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/69dc32ba-ecdd-4cec-908c-e8a2401b8018)
